### DISCUSSION TITLE
Ai image generation

### DISCUSSION THREAD

**Discussion by nightfury-toothless** [community/community#68190] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello
I am making AI model for face editing tools 
I am using cycleGAN,
but I have some issue with image clarify.

Please help me.

**Comment by tp1845** [comment]
Hello, there.
maybe, that doesn't work for your tools,
because CycleGAN model is used to transfer characteristic of one image to another.
You may use pixel2pixel for that.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by HeraclesG** [comment]
Pixel2pixel is better than CycleGAN.
